libs/dh/globalization/feature-language-picker/src/lib/language-button/dh-language-button.component.html

```diff
@@ -0,0 +1,22 @@
+<!--
+@license
+Copyright 2021 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<watt-button
+  type="text"
+  [disabled]="isDisabled"
+  (click)="languageSelect.emit()"
+  >{{ language | uppercase }}</watt-button
```

Comment:
suggestion: `uppercase` shouldn't be necessary because button's text is uppercased with CSS.

Reply:
The languages are uppercased to `DA` and `EN` for accessibility.

Reply:
If it's not an elaborate explanation, can you share some words on the accessibility part? Does capitalization have an effect on how screen readers read the text?

Reply:
Yes, screen readers will read it differently and the letters get a different meaning semantically, making it more clear. We should also add descriptive titles but I didn't want to add that before we have more content and a better grip on Transloco and the structure of our translation texts.